ci/scripts/driver_weekly.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+set -eux
+
+##############################################################################################
+#
+# Script description: Top level driver script for checking PR
+#                     ready for CI regression testing
+#
+# Abstract:
+#
+# This script runs the high resolution cases found in $HOMEgfs/ci/cases/weekly
+# from the develop branch for the global-workflow repo that is intended to run on a weekly basis
+# from a cron job. When ran it will clone and build a new branch from the EMC's global-workflow and
+# and create a pr using GitHub CLI by moving and replacing the yaml case files in
+# ${HOMEgfs}/ci/cases/weekly to {HOMEgfs}/ci/cases/pr.  Then at point it can simply also add the
+# the requisite labels so that current BASH CI framework can run these cases.
+#######################################################################################i######
+
+#################################################################
+# TODO using static build for GitHub CLI until fixed in HPC-Stack
+#################################################################
+export GH=${HOME}/bin/gh
+export REPO_URL=${REPO_URL:-"https://github.com/NOAA-EMC/global-workflow.git"}
+
+################################################################
+# Setup the reletive paths to scripts and PS4 for better logging 
+################################################################
+HOMEgfs="$(cd "$(dirname  "${BASH_SOURCE[0]}")/../.." >/dev/null 2>&1 && pwd )"
+scriptname=$(basename "${BASH_SOURCE[0]}")
+echo "Begin ${scriptname} at $(date -u)" || true
+export PS4='+ $(basename ${BASH_SOURCE})[${LINENO}]'
+
+#########################################################################
+#  Set up runtime environment varibles for accounts on supproted machines
+#########################################################################
+
+source "${HOMEgfs}/ush/detect_machine.sh"
+case ${MACHINE_ID} in
+  hera | orion)
+    echo "Running Automated Testing on ${MACHINE_ID}"
+    source "${HOMEgfs}/ci/platforms/${MACHINE_ID}.sh"
+    ;;
+  *)
+    echo "Unsupported platform. Exiting with error."
+    exit 1
+    ;;
+esac
+
+######################################################
+# setup runtime env for correct python install and git
+######################################################
+set +x
+source "${HOMEgfs}/ush/module-setup.sh"
+module use "${HOMEgfs}/modulefiles"
+module load "module_gwsetup.${MACHINE_ID}"
+set -x
+
+#########################################################
+# Create a new branch from develop and move yaml files
+#########################################################
+branch="weekly_ci_$(date +%A_%b_%Y)"
```

Comment:
`date +%A_%b_%Y` yeilds `WeekDayName_Mon_YYYY`.  It should contain `day` as `DD`.  I suggest the form YYYY-MM-DD 
```suggestion
branch="weekly_ci_$(date +%Y%m%d)"
```

Reply:
Personally I like the words, but ok.

Reply:
Words are fine, but remember this is weekly test, so the branch for all 4 weeks (assume Friday) of September 2023 will be:
`Friday_Sept_2023`.
It does not tell you which specific date it is: Sept 7, 14, 21 or 28.